app/controllers/users_controller.rb

```diff
@@ -1,10 +1,10 @@
 # frozen_string_literal: true
 
 class UsersController < ApplicationController
-  load_and_authorize_resource
+  load_and_authorize_resource except: %i[edit update]
 
   def index
-    @users = User.all
+    @users = User.order(cell: :asc)
   end
 
   def show; end
```

Comment:
(can't add comment for edit) but prolly good to redirect user away from the change password form if `params[:id]` is not current user signed in